src/components/common/form-row/FormRow.tsx

```diff
@@ -55,24 +59,21 @@ type Props = {
   dense?: boolean;
   id: string;
   label: string;
+  multiple?: boolean;
```

Comment:
this is only to remove the bottom padding if we are only displaying one form row